map/src/drawer/components/header/HeaderInfo.jsx

```diff
@@ -0,0 +1,134 @@
+import {Box, Typography} from "@mui/material";
+import React, {useContext, useEffect, useState} from "react";
+import AppContext, {toHHMMSS} from "../../../context/AppContext";
+import SearchInfo from "./SearchInfo";
+
+export default function HeaderInfo() {
+
+    const ctx = useContext(AppContext);
+
+    const [textInfo, setTextInfo] = useState('');
```

Comment:
2. This looks too much code and logic and ifs for Generic Header. As we think that Header is controlled by **Drawer** we need to make sure that ll computation of details and function should stay in **DrawerGroupItem**. Suggested structure would be 
```
 const [headerInfo, setHeaderInfo] = useState({
})
{
   tracks: { 'text':'..'},
   weather: { 'text':'...'},
   search: { 'text':'...'},
   welcome: { 'text':'...'}
}
```

Here we need to make sure that we can orchestrate content